lib/helpers.rb

```diff
@@ -15,8 +15,12 @@ def thread
     @thread ||= CommentThread.find(params[:thread_id])
   end
 
-  def comment
-    @comment ||= Comment.find(params[:comment_id])
+  def comment(comment_id=nil, sync_child_count=false)
+    comment_id ||= params[:comment_id]
+    @comment ||= Comment.find(comment_id)
+    syncd_child_count = @comment.sync_child_count(sync_child_count)
```

Comment:
See comment above.  Rather than the sync_child_count param, move logic to a new method (as documented above) and just call it where it is needed.